### DISCUSSION TITLE
how to silence the server-notifications?

### DISCUSSION THREAD

**Discussion by fommil** [joaotavora/eglot#1326] [Q&A]
Hi all,

I don't like the way `server-notification` can pop up in my minibuffer. I'd rather they stay entirely within the realm of the `*EGLOT*` buffer. Is there a way disable notifications being echoed to my minibuffer? I cannot find anything relevant in https://www.gnu.org/software/emacs/manual/html_node/eglot/Eglot-Variables.html

**Comment by joaotavora** [comment]
This is a duplicate of #1325.  Please avoid spawning multiple threads at once..

**Reply by fommil** [reply]
This isn't a duplicate of the other ticket, any LSP server (regardless of language) can send server notifications and I'm interested in silencing them so that they don't go to the minibuffer.

**Reply by fommil** [reply]
@joaotavora apologies, but I think there is a misunderstanding about what this question is asking. Would you like me to ask it again or can we reopen this?

This is specifically about disabling the echoing any kind of message from the server to the minibuffer. Not about rust diagnostics in particular, that's just an example of the kind of thing that shows up in the minibuffer.

**Reply by joaotavora** [reply]
OK, set `eglot-report-progress` to `nil`

**Reply by fommil** [reply]
ah, awesome, I didn't know about this setting I'll have a look at the source code in that area to see what else I'm missing out on, thank you!
